go-controller/cmd/ovn-kube-util/app/readiness-probe.go

```diff
@@ -103,11 +109,29 @@ func ovnSBDBReadiness(target string) error {
 }
 
 func ovnNorthdReadiness(target string) error {
-	// checking version works as it connects to northd and returns the version
-	// if northd isn't ready, version may fail to return
-	_, _, err := util.RunOVNAppctlWithTimeout(5, "-t", target, "version")
+	stdout, _, err := util.RunOVNAppctlWithTimeout(5, "-t", target, "status")
 	if err != nil {
-		return fmt.Errorf("failed to get version from %s: (%v)", target, err)
+		return fmt.Errorf("failed to get status from %s: (%v)", target, err)
+	} else if strings.HasPrefix(stdout, "Status") {
+		output := strings.Split(stdout, ":")
+		status := strings.TrimSpace(strings.Trim(output[1], "\n"))
```

Comment:
@girishmg TrimSpace trims newline, so I think you can just do `strings.TrimSpace(output[1])` ?

Reply:
ACK